### PR TITLE
Validated input size fix

### LINKED ISSUE #1287: input changes width and height when invalid
Switching input state from default to invalid causes it to change width and height (as far as I can see because of the font weight change). This looks terrible.

Same thing for the "valid" state.

input-bug mov

### PR DESCRIPTION
**Pull request by anna-wro**
Another way would be to use [`-webkit-text-stroke`](http://caniuse.com/#feat=text-stroke) - WDYT?

screen shot 2017-09-14 at 13 39 59
screen shot 2017-09-14 at 13 39 44
screen shot 2017-09-14 at 13 40 06

Fixes #1287

### CODE REVIEW

File: src/components/form-elements/_text-input.scss [hunk 1]
```diff
@@ -103,12 +103,18 @@ $includeHtml: false !default;
 
     &--valid {
       color: $inputFocusValidBorderColor;
-      font-weight: $fontWeightBold;
+      text-shadow: -0.2px -0.2px 0 $inputFocusValidBorderColor,
+      0.2px -0.2px 0 $inputFocusValidBorderColor,
+      -0.2px 0.2px 0 $inputFocusValidBorderColor,
+      0.2px 0.2px 0 $inputFocusValidBorderColor;
```
Comment (kdzwinel) [inline src/components/form-elements/_text-input.scss]: Yeah, let's try `text-stroke` :) Support looks OK to me.

Comment (anna-wro) [reply src/components/form-elements/_text-input.scss]: changed!

File: src/components/form-elements/_text-input.scss [hunk 2]
```diff
@@ -103,12 +104,12 @@ $includeHtml: false !default;
 
     &--valid {
       color: $inputFocusValidBorderColor;
-      font-weight: $fontWeightBold;
+      -webkit-text-stroke: 0.5px $inputFocusValidBorderColor;
```
Comment (kdzwinel) [inline src/components/form-elements/_text-input.scss]: do we need a vendor prefix? I thought sass is adding it by itself?

Comment (anna-wro) [reply src/components/form-elements/_text-input.scss]: It didn't work without it :/

Comment (kdzwinel) [reply src/components/form-elements/_text-input.scss]: hm, and does it work in firefox/safari/etc.? we will probably also need `-moz` etc.?

File: src/components/form-elements/_text-input.scss [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// sass-lint:disable no-vendor-prefixes
```
Comment (kdzwinel) [inline src/components/form-elements/_text-input.scss]: can you disable this rule only for single lines that need it, not the whole file?

Comment (anna-wro) [reply src/components/form-elements/_text-input.scss]: sure

File: src/components/form-elements/_text-input.scss (hunk 2 above)
Comment (anna-wro) [reply src/components/form-elements/_text-input.scss]: I checked in both of them and it works

**Comment by kdzwinel** [comment]
Thanks for fixing it 🤘

**Pull request merged**